astronomer/providers/google/cloud/operators/dataproc.py

```diff
@@ -267,3 +272,103 @@ def execute_complete(self, context: Dict[str, Any], event: Optional[Dict[str, st
                 return event["job_id"]
             raise AirflowException(event["message"])
         raise AirflowException("No event received in trigger callback")
+
+
+class DataprocUpdateClusterOperatorAsync(DataprocUpdateClusterOperator):
+    """
+    Updates a cluster in a project.
```

Comment:
Can you add more Description, so that it looks good in documentation

Reply:
Updated